stacker/blueprints/base.py

```diff
@@ -462,6 +463,16 @@ def render_template(self):
         version = hashlib.md5(rendered).hexdigest()[:8]
         return (version, rendered)
 
+    def to_json(self):
+        """Render the blueprint and return the template in json form."""
+
+        variables = []
+        for k in self.get_parameter_definitions():
+            variables.append(Variable(k, 'unused_value'))
```

Comment:
Doesn't this break in the case of the variable not accepting a string type?

Reply:
`get_parameter_definitions` exclusively returns CFN parameters, so they will always take a string.